app/views/all_casa_admins/dashboard/show.html.erb

```diff
@@ -13,13 +13,17 @@
       <tr>
         <th>Name</th>
         <th>Created At</th>
+        <th>User Count</th>
+        <th>Case Contact Count</th>
       </tr>
       </thead>
       <tbody>
       <% @organizations.each do |organization| %>
         <tr>
           <td><%= link_to organization.name, all_casa_admins_casa_org_path(organization) %></td>
           <td><%= I18n.l(organization.created_at, format: :full, default: nil) %></td>
+          <td><%= organization.user_count %></td>
```

Comment:
It would be nice to add to a test to assert that these lines show up in the view (this PR or a later PR is fine)

Reply:
Yeah, agreed. The only reason I didn't is because there aren't any specs (that I could find) that checked that view in that much detail. Are there any capybara or similar specs somewhere in the codebase otherwise I can create a Cypress spec?

Reply:
No worries, I've found the system spec for the dashboard

Reply:
Yay! :)

Reply:
And added those tests :)